mmaction/datasets/pipelines/augmentations.py

```diff
@@ -351,6 +352,8 @@ def __init__(self, scales, mode='range', **kwargs):
                              f'but got {self.mode}')
         self.scales = scales
         self.kwargs = kwargs
+        warnings.warn('This augmentation is not used in our experiments, '
```

Comment:
Reason for this warn?

Reply:
This aug is not used & we don't know if it works properly in the pipeline.

Reply:
Okay, we can deprecate it in next version and remove it in the next next version

Reply:
https://github.com/open-mmlab/mmaction2/pull/787

Reply:
I'll change it here, in this PR.